File: Functions/Financial Calculator.py
# Sasha Rinitra, Update Financial Calculator Python

# Write a print statement telling the user what the program is (Budget Calculator)
print("Welcome to Financial Calculator!")

def user(type):
    return float(input(f"How much is your monthly {type}? \n"))
    
income = user("income")
rent = user("rent")
utilities = user("utilities")
groceries = user("groceries")
transportation = user("transporation")

savings = income*0.1
expenses = rent+utilities+groceries+transportation+savings

def info(income, amount, type):
    percent_type = float(amount/income*100)
    print(f"You spend ${amount:.2f} on {type} and that is {percent_type:.1f}% of your income")

info(income, rent, "rent")
info(income, utilities, "utilities")
info(income, groceries, "groceries")
info(income, transportation, "transportation")
info(income, savings, "savings")

print(f"You saved ${savings:.2f} and that is 10% of your income")

File: Functions/Financial_Calculator.c
// Sasha Rinitra, Financial Calculator C

#include <stdio.h>

char welcome[] = "Welcome to Budget Calculator!";
float income, rent, utilities, groceries, transportation;


float user(char type[]){
    float amount;
    printf("How much is your %s amount monthly?\n", type);
    scanf("%f", &amount);
    return amount;
}

void info(float income, float amount){
    float percent = (amount/income)*100;
    printf("You spend $%.2f on rent which is %.2f%% your income\n", amount, percent);
}

int main(void){
    printf("%s\n", welcome);
    
    income = user("income");
    rent = user("rent");
    utilities = user("utilities");
    groceries = user("groceries");
    transportation = user("transporation");

    float savings = income/10;
    float expenses = income - (rent+utilities+groceries+transportation+savings);
    float expenses_percent = (expenses/income)*100;

    info(income, rent);
    info(income, utilities);
    info(income, groceries);
    info(income, transportation);
    info(income, savings);
    printf("You have $%.2f for savvings which is 10%% of your income\n", savings);
    printf("You have $%.2f left for spending which is %.2f%% of your income\n", expenses, expenses_percent);

    return 0;
}